test/integration/targets/apt/tasks/apt.yml

```diff
@@ -437,3 +437,48 @@
   file:
     path: /usr/sbin/policy-rc.d
     state: absent
+
+# https://github.com/ansible/ansible/issues/65325
+- name: install hello with apt (to test allow_change_held_packages option)
+  apt:
+    name: hello
+    state: present
+    allow_change_held_packages: yes
+  register: apt_result
```

Comment:
I don't see where `hello` is added to the list of held packages before running a task using this option.

Reply:
I believe the general outline of these tests should be:
- install an older version of a package
- add a hold to a package with `apt-mark hold [package]`
- verify package is on the hold list with `apt-mark showhold`
- try to update the package and ensure it did not change
- try to update the package with `allow_change_held_packages=True` and ensure it did change
- try to update the package again with `allow_change_held_packages=True` and ensure it did not change
- remove the package and the hold in a handler

Reply:
Yes. You are right. The question I was facing was where to get a old package of something. So I decided to at least to check that I did not introduce any BC break since I was not able to find any integration tests about held packages at all (in the apt part of the integration tests). Found them of course now in the dpkg_selections part and will use the old hello package from s3 they used there.